.gitignore

```diff
@@ -264,8 +264,6 @@ images/runtime/dotnetcore/*/Dockerfile
 images/runtime/dotnetcore/*/buster.Dockerfile
 images/runtime/dotnetcore/*/stretch.Dockerfile
 images/runtime/python/*/Dockerfile
-images/runtime/python/*/buster.Dockerfile
```

Comment:
These lines shouldn't be removed -- my original comment was intended for you to remove the changes you originally made, and add the following line underneath these lines:

```
images/runtime/python/*/bullseye.Dockerfile
```

The only change in this `.gitignore` file should be the addition of this `bullseye.Dockerfile` line I mentioned above to be right underneath line 268 (which is the `stretch.Dockerfile` line for `python`)

Reply:
yah, this file was committed unintentionally. Now, I have fixed this as you suggested.